scripts/HPC_chatbot_preprocessor/parsed_mds/generic/apptainer/apptainer_paragraph_1.txt

```diff
@@ -0,0 +1,42 @@
+Apptainer (formally known as Singularity)
+What is Apptainer?
+Apptainer is an open-source computer program that performs
+operating-system-level virtualization (also known as containerisation).
+One of the main uses of Apptainer is to bring containers and
+reproducibility to scientific computing and the high-performance
+computing (HPC) world. Using Apptainer/Singularity containers,
+developers can work in reproducible environments of their choosing and
+design, and these complete environments can easily be copied and
+executed on other platforms.
+For more general information about the use of Apptainer, please see the
+official documentation at https://apptainer.org/docs/.
+This documentation only covers aspects of using Apptainer on the
+HPC-UGent infrastructure infrastructure.
+Restrictions on image location
+Some restrictions have been put in place on the use of Apptainer. This
+is mainly done for performance reasons and to avoid that the use of
+Apptainer impacts other users on the system.
+The Apptainer/Singularity image file must be located on either one of
+the scratch filesystems, the local disk of the workernode you are using
+or /dev/shm. The centrally provided apptainer command will refuse to
+run using images that are located elsewhere, in particular on the
+$VSC_HOME, /apps or $VSC_DATA filesystems.
+In addition, this implies that running containers images provided via a
+URL (e.g., shub://... or docker://...) will not work.
+If these limitations are a problem for you, please let us know via hpc@ugent.be.
```

Comment:
if they are a problem, just copy the config file, and pass the modfied one via `--conifg ...` ;)